docs/react-wiki-archive/Fluent-UI-React-v8-Sunsetting.md

```diff
@@ -0,0 +1,43 @@
+# Fluent v8 sunsetting bug process
+
+This document outlines the updated standards for addressing bugs on Fluent v8 in its sunsetting phase. This covers the bar for addressing bugs until v8 is fully deprecated in July 2026, at which point only critical / sev0 bugs will be addressed on a case-by-case basis.
+
+## Timeline
+
+- July 2025: Fluent v8 began its sunsetting phase in as [announced in our newsletter](https://outlook.office.com/newsletters/newsletters/a2e6ab48b33a3341a4ec2ac99108eebb23). Bug handling is described in this document.
+- July 2026: v8 will be fully in maintenance mode, only critical bugs will be considered on a case-by-case basis.
```

Comment:
it will reach EOL, branching out according to our SG https://github.com/microsoft/fluentui/blob/master/docs/react-v9/contributing/rfcs/shared/repo-organization.md#branching-strategy